docs/configuration.md

```diff
@@ -144,6 +146,8 @@ Values set in the configuration file always overwrite values set in the strategy
 * `use_sell_signal` (ask_strategy)
 * `sell_profit_only` (ask_strategy)
 * `ignore_roi_if_buy_signal` (ask_strategy)
+* `ignore_buying_expired_candle`
+* `ignore_buying_expired_candle_after`
```

Comment:
should we allow this to be set in the configuration as well (otherwise this is the wrong place to document these settings).

If the answer is yes, then you must add them to the attributes in `strategy_resolver.py`.

Reply:
My idea would be that this is closely tied to the timeframe, as it has a (somewhat) direct relationship with it. The timeframe is something that belongs with the strategy I believe so that's the reason I put it here. Do you think it should be in the configuration @xmatthias ?

Reply:
while timeframe CAN be set by the strategy, you can also have it in the configuration - which will then override what you had in the strategy.
I don't use it myself, but having it in the configuration is the default for all sample configurations iirc.

so while timeframe CAN come from the strategy, it can also come from the strategy - so allowing it in both places might be a consequence.

Reply:
I moved them to strategy_resolver.py, but I'm not sure I did it right, so please check and let me know if it needs to be done differently!